numpy/distutils/cpuinfo.py

```diff
@@ -46,8 +46,7 @@ def command_by_line(cmd, successful_status=(0,), stacklevel=1):
                            stacklevel=stacklevel+1)
     if not ok:
         return
-    for line in output.splitlines():
-        yield line.strip()
+    yield from (line.strip() for line in output.splitlines())
```

Comment:
I don't think this type of transformation (introducing a generator comprehension) is a helpful one, but will let another reviewer decide.

Reply:
the new code does not seem to be any cleaner than the old code, so I would be +1 on leaving it alone